packages/intrepid2/src/Discretization/Basis/Intrepid2_HCURL_TET_In_FEM.hpp.orig

```diff
@@ -0,0 +1,341 @@
+// @HEADER
```

Comment:
@mperego, I think you might have included this file by accident.  (It has some git merge conflict markings in it.)

Reply:
Argh! Good catch